UPGRADING.md

```diff
@@ -0,0 +1,57 @@
+## Upgrading to v1.0
+
+In an initializer add the following
+```ruby
+Money.configure do |config|
+  config.legacy_default_currency!
+  config.legacy_deprecations!
+  config.legacy_json_format!
+  #...
+end
+```
+
+Remove each legacy setting making sure your app functions as expected.
```

Comment:
please do 👍

Reply:
Haven't been able to find time for this, nor will I be able to in the next couple of weeks. 

Roughly speaking, what I had in mind as an upgrade guide if someone wants to take it on:
1. use [the included Rubocop cops](https://github.com/Shopify/money#rubocop) to autocorrect what can be found with static analysis
2. record deprecations using https://github.com/Shopify/deprecation_toolkit/ for anything not covered by 1 to prevent new deprecations to be included.